servicecontrol/creating-config-file.md

```diff
@@ -142,4 +142,9 @@ net stop particular.servicecontrol
 
 * `ServiceBus/AuditLogQueue` (string)
 
-  The audit queue name to use for forwarding audit messages. This only works if `ServiceControl/ForwardAuditMessages` is true. Default `<AuditQueue>.log`
\ No newline at end of file
+  The audit queue name to use for forwarding audit messages. This only works if `ServiceControl/ForwardAuditMessages` is true. Default `<AuditQueue>.log`
+
+
+* `ServiceControl/MaxBodySizeToStore` (int)
+
+  By default we only store bodies of audit messages that are smaller than 100Kb. Increase this number to store messages with larger bodies.  Default `102400`
```

Comment:
> so if there is nothing wrong with that why do we have a limit at all?

Good point, @petersgiles the same way you did in SI, we should explain here why such limit exists